backend/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM php:7.4-fpm-alpine
```

nit: Do we need `fpm` if we run artisan?
```suggestion
FROM php:7.4-alpine
```

---

I will make a test to see if it's working without fpm

---

You were right! I made another change, Thank you!